src/analysis/AppLauncher.ts

```diff
@@ -376,11 +440,6 @@ const ApplicationLauncher = _.flow(
 );
 
 export const navPaths = [
-  {
```

Comment:
So this is some legacy stuff that is no longer needed?

Reply:
Yeah, it isn't referenced anywhere in the UI as a link